examples/helia-browser-react-verified-fetch/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "helia-browser-react-verified-fetch",
```

Maybe a simpler name?

```suggestion
  "name": "helia-verified-fetch",
```

---

I'd like to keep `browser` in the name. That's a very important distinction for users.

---

Ok, leave `browser` in then.